app/models/voice_response.rb

```diff
@@ -0,0 +1,24 @@
+class VoiceResponse < ApplicationRecord
+    
+    def initialize(call)
+        @call = call
+        case @call
+        when BeginCall then @response = Twilio::TwiML::VoiceResponse.new do |r| 
```

Comment:
Don't use `then`. Just start your block on the next line.
```ruby
when BeginCall
  @response = Twilio::TwiML::VoiceResponse.new do |r|
```

Use helpful names for variables. `r` => `response`